src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/List.js

```diff
@@ -513,6 +515,7 @@ class List extends React.Component<Props> {
             adapterOptions,
             adapters,
             copyable,
+            customButtons,
```

Comment:
not sure about this param and its place not sure if the buttons should be adapter specific. Here @nnatter had I think a better overview what is consistence.

At current state I personally would call it just `buttons` don't see a usage to add a `custom` prefix here.